solutions/devsprint-adriano-song-1/RealEstateApp/Screens/LoginView/LoginView.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  LoginView.swift
+//  RealEstateApp
+//
+//  Created by Fabio Martinez on 17/02/22.
+//
+
+import Foundation
+import UIKit
+
+
+final class LoginView: UIView {
+    
+    private var accountLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Login with you account"
+        label.textColor = .black
+        label.font = UIFont.boldSystemFont(ofSize: 16)
+        label.textAlignment = .center
+        return label
+    }()
+    
+    private let emailTextField: UITextField = {
+        let textField = UITextField()
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        textField.layer.cornerRadius = 10
+        textField.placeholder = "Type your E-mail"
+        textField.autocorrectionType = .no
+        textField.layer.borderWidth = 1
+        textField.layer.borderColor = UIColor.lightGray.cgColor
+        textField.textAlignment = .center
+        return textField
+    }()
+    
+    private let passwordTextField: UITextField = {
```

Comment:
Please make passwordTextField a part of ContactFormView component (see Implementar ContactFormView task)